scripts/fhd_batch_convert.py

```diff
@@ -16,8 +16,8 @@
 def parse_range(string):
     m = re.match(r'(\d+)(?:-(\d+))?$', string)
     if not m:
-        raise ArgumentTypeError("'" + string + "' is not a range of numbers."
-                                " Expected forms like '0-5' or '2'.")
+        raise argparse.ArgumentTypeError(
+            "'{}' is not a range of numbers. Expected forms ".format(string))
```

Comment:
The`Expected forms like '0-5' or '2'` bit of this now just reads `Expected forms`. Should the `'0-5' or '2'` still be on line 20?

Reply:
oops, good catch.